Resources/Prototypes/Entities/Structures/Doors/Airlocks/base_structureairlocks.yml

```diff
@@ -131,5 +131,8 @@
   - type: DynamicPrice
     price: 150
   - type: AccessReader
+  - type: Tag
```

Comment:
Probably not needed, maybe good to have anyway? I could change collision to be an EntityWhitelist instead of tags and then just use the Component